apps/condo/migrations/20230223172911-0226_manual_create_BankCostItem_records.js

```diff
@@ -0,0 +1,135 @@
+exports.up = async (knex) => {
+    await knex.raw(`
+        BEGIN;
+        --create BankCategory
```

Comment:
This comment does not provides additional information.
We clearly see lines containing `INSERT INTO "BankCategory"` statement, and then lines containing `INSERT INTO "BankCostItem"`.
No visual differentiation is needed here IMO.